cli/testflinger_cli/client.py

```diff
@@ -56,14 +57,16 @@ def get(self, uri_frag, timeout=15, headers=None):
         uri = urllib.parse.urljoin(self.server, uri_frag)
         try:
             req = requests.get(uri, timeout=timeout, headers=headers)
-        except requests.exceptions.ConnectionError:
-            logger.error("Unable to communicate with specified server.")
-            raise
-        except IOError:
-            # This should catch all other timeout cases
-            logger.error(
-                "Timeout while trying to communicate with the server."
-            )
+            self.error_count = 0
+        except (IOError, requests.exceptions.ConnectionError) as exc:
+            self.error_count += 1
+            if self.error_count % 10 == 0:
```

Comment:
The divisor is hard-coded to 10 here, as well as the tests. I think it will give us a lot more flexibility if we could get this value from the configuration file.

Reply:
Thanks for the feedback! I've added a config setting, documentation, and made the default 3.